tools/refguide_check.py

```diff
@@ -861,69 +974,55 @@ def main(argv):
-            sys.stderr.flush()
+    if args.doctests or not args.rst:
+        print("Running checks for %d modules:" % (len(modules),))
+        for module in modules:
+            if dots:
+                sys.stderr.write(module.__name__ + ' ')
+                sys.stderr.flush()
 
-        all_dict, deprecated, others = get_all_dict(module)
-        names = names_dict.get(module.__name__, set())
+            all_dict, deprecated, others = get_all_dict(module)
+            names = names_dict.get(module.__name__, set())
 
-        mod_results = []
-        mod_results += check_items(all_dict, names, deprecated, others, module.__name__)
-        mod_results += check_rest(module, set(names).difference(deprecated),
-                                  dots=dots)
-        if args.doctests:
-            mod_results += check_doctests(module, (args.verbose >= 2), dots=dots,
-                                          doctest_warnings=args.doctest_warnings)
+            mod_results = []
+            mod_results += check_items(all_dict, names, deprecated, others,
+                                       module.__name__)
+            mod_results += check_rest(module, set(names).difference(deprecated),
+                                      dots=dots)
+            if args.doctests:
+                mod_results += check_doctests(module, (args.verbose >= 2), dots=dots,
+                                              doctest_warnings=args.doctest_warnings)
 
-        for v in mod_results:
-            assert isinstance(v, tuple), v
+            for v in mod_results:
+                assert isinstance(v, tuple), v
 
-        results.append((module, mod_results))
+            results.append((module, mod_results))
 
-    if dots:
-        sys.stderr.write("\n")
-        sys.stderr.flush()
-
-    if not args.skip_tutorial:
-        base_dir = os.path.join(os.path.abspath(os.path.dirname(__file__)), '..')
-        tut_path = os.path.join(base_dir, 'doc', 'source', 'tutorial', '*.rst')
-        print('\nChecking tutorial files at %s:' % os.path.relpath(tut_path, os.getcwd()))
-        for filename in sorted(glob.glob(tut_path)):
             if dots:
                 sys.stderr.write('\n')
-                sys.stderr.write(os.path.split(filename)[1] + ' ')
                 sys.stderr.flush()
 
-            tut_results = check_doctests_testfile(filename, (args.verbose >= 2),
-                    dots=dots, doctest_warnings=args.doctest_warnings)
-
-            def scratch(): pass        # stub out a "module", see below
-            scratch.__name__ = filename
-            results.append((scratch, tut_results))
+            all_dict, deprecated, others = get_all_dict(module)
 
+    if args.rst:
+        base_dir = os.path.join(os.path.abspath(os.path.dirname(__file__)), '..')
+        rst_path = os.path.relpath(os.path.join(base_dir, args.rst))
+        if os.path.exists(rst_path):
+            print('\nChecking files in %s:' % os.path.relpath(rst_path, os.getcwd()))
```

Comment:
```suggestion
            print('\nChecking files in {}:'.format(rst_path))
```
You already called `relpath` on line 1010, no need to do so a second time

Reply:
ok